linux/powershell/PSCloudShellUtility/PSCloudShellUtility.psm1

```diff
@@ -1885,13 +1885,15 @@ function Get-PackageVersion() {
     
     # Enumerate all APT packages with versions
     $packages = New-Object -TypeName System.Collections.ArrayList
-    apt list --installed 2> /dev/null | % { 
-        Write-Verbose "Apt: $_"
-        if ($_ -match "([^/]*)/[^ ]* ([^ ]*)") { 
-            $p = New-PackageInfo -Name $matches[1] -Version $matches[2] -Type "Apt"
-            $null = $packages.Add($p)
-        }
-    }
+
+    # TODO - find the regular expression to seperate the package name from the package version
```

Comment:
Remove commented out lines?

Reply:
Hmm, that was intentional because I would want this added in sometime in the future.